src/libraries/System.Threading.Tasks.Parallel/src/System/Threading/Tasks/Parallel.cs

```diff
@@ -346,7 +346,9 @@ public static void Invoke(ParallelOptions parallelOptions, params Action[] actio
                     // threw an exception.  We let such exceptions go completely unhandled.
                     try
                     {
+#pragma warning disable CA1416 // Validate platform compatibility, proper handling needed
```

Comment:
This needs tracking issue

Reply:
issue: https://github.com/dotnet/runtime/issues/44605